<launch>
    <arg name="is_sim" default="$(optenv SIMULATION true)" />
    <arg name="robot1_name" default="recbot0" />
    <arg name="robot2_name" default="grizzly0" />
    
    <include file="$(find waypoint_planner)/launch/evaluate_policies.launch">
        <arg name="robot_name" value="$(arg robot1_name)" />
    </include>

    <include file="$(find waypoint_planner)/launch/evaluate_policies.launch">
        <arg name="robot_name" value="$(arg robot2_name)" />
    </include>

</launch>